tests/integration/StorageBucketTest.php

```diff
@@ -11,8 +11,11 @@ final class StorageBucketTest extends TestCase
 	public function setup(): void
 	{
 		parent::setUp();
-		$authHeader = ['Authorization' => 'Bearer '.'<service_role>'];
-		$this->client = new  \Supabase\Storage\StorageClient('https://<project_ref>.supabase.co', $authHeader);
+		$dotenv = \Dotenv\Dotenv::createUnsafeImmutable(dirname(__DIR__, 2), '.env.test');
```

Comment:
We should use a path here e.g. `__DIR__ . '/../../env.test';` vs `dirname()`

Reply:
Done